src/osemgrep/tests/Test_target_selection.ml

```diff
@@ -2,45 +2,58 @@
    Test target selection on git repos with osemgrep.
 *)
 
+open Printf
 module T = Alcotest_ext
 
 let with_git_repo (files : Testutil_files.t list) func =
   Testutil_files.with_tempfiles_verbose files (fun path ->
       Testutil_files.with_chdir path (fun () ->
           Git_wrapper.init ();
-          Git_wrapper.add [ Fpath.v "." ];
+          Git_wrapper.add ~force:true [ Fpath.v "." ];
           Git_wrapper.commit "Add all the files";
           func ()))
 
-let osemgrep_ls () =
-  (* TODO: see/reuse what's being done in Test_osemgrep.ml *)
-  ()
+let run_osemgrep caps argv =
+  printf "RUN %s\n%!" (argv |> Array.to_list |> String.concat " ");
+  CLI.main caps argv
+
+let osemgrep_ls caps =
+  let exit_code =
+    run_osemgrep caps [| "semgrep"; "scan"; "--experimental"; "--x-ls"; "." |]
+  in
+  Alcotest.(check int) "exit code" 0 (Exit_code.to_int exit_code)
+
+let concat_lines lines = String.concat "\n" lines ^ "\n"
```

Comment:
Same nit re: Common.unlines

Reply:
I'm going to keep this because it saves a documentation lookup, `String.concat` being commonly used and part of the OCaml standard library.